src/less/styles.less

```diff
@@ -169,41 +168,3 @@
 }
  
 
-
-
-//  .legend {
```

Comment:
This legend is useful, as it defines what the node color progression means.  Can we add it back?